src/Aurora.Infrastructure/Services/SearchDataService.cs

```diff
@@ -23,26 +22,28 @@ public async Task StoreRequest(SearchRequestDto request)
         {
             List<SearchRequest> existingRequests = await GetExistingFor(request);
 
+            List<SearchRequest> newRequests = new List<SearchRequest>();
             foreach (var webSite in request.Websites)
             {
                 foreach (var option in request.SearchOptions)
                 {
                     if (existingRequests.Where(x => x.Website == webSite && x.ContentOption == option).Any() == false)
```

Comment:
to hard to understand, perhaps move the logic to methos? what this logical expression is checking

Reply:
moved to method